exporter/coralogixexporter/go.mod

```diff
@@ -19,7 +19,6 @@ require (
 )
 
 require (
-	cloud.google.com/go/compute/metadata v0.2.4-0.20230617002413-005d2dfb6b68 // indirect
 	github.com/cenkalti/backoff/v4 v4.2.1 // indirect
```

Comment:
Some of these go.mod changes appear unrelated

Reply:
I believe these all came from running `make gotidy`. Can see the changes isolated in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/26297/commits/eb94c8efadc3e3cb9698fd38065da01deb823c61. Is there another way to get the gopsutil dep update synced without these changes?